project/plugin.sbt

```diff
@@ -1,5 +1,3 @@
-libraryDependencies += "org.scala-sbt" %% "scripted-plugin" % sbtVersion.value
```

Comment:
ScriptedPlugin is now part of SBT and doesn't require additional plugin registration